code/parachain/runtime/composable/src/fees.rs

```diff
@@ -0,0 +1,21 @@
+use common::fees::{ForeignToNativePriceConverter, PriceConverter};
+use composable_traits::{
+	currency::{Rational64, Unit},
+	rational,
+};
+
+use primitives::currency::CurrencyId;
+
+pub struct WellKnownForeignToNativePriceConverter;
+impl ForeignToNativePriceConverter for WellKnownForeignToNativePriceConverter {
+	fn get_ratio(asset_id: CurrencyId) -> Option<Rational64> {
+		match asset_id {
+			CurrencyId::xcDOT => Some(rational!(2143 / 1_000_000)),
```

Comment:
where does this number come from?

Reply:
just something not to spam network. created task to find out and set release values https://app.clickup.com/t/8669gtt7j